app/forms/FormsHelper.scala

```diff
@@ -36,7 +36,7 @@ object FormsHelper {
   def populatedForm(inputStanza: DataInput, path: String, answer: Option[String]): Form[_] =
     inputStanza match {
       case _: DateInput => populateSubmittedDateAnswerForm(answer)
-      case _: Input | _: Question => populateSubmittedTextAnswerForm(path -> nonEmptyText, path, answer)
+      case _: Input | _: Question | _: Sequence => populateSubmittedTextAnswerForm(path -> nonEmptyText, path, answer) // TODO - Temporary solution whilst rendering sequence
```

Comment:
think this will also need adding to bindFormData above

Reply:
This will be done as part of the submission task.